### ISSUE TITLE
How to install POGO-POPULATE

### ISSUE THREAD

**Issue by LUCKASSs** [maierfelix/POGOserver#281]
Hello guys,
I try to install `[pogo-populate](https://github.com/JoshGlue/pogo-populate)` on my server , but I get always an error when starting the server.
So any of you can explain me steps by steps how to install this please ?
I have posted an issue on this github but seems to be outdated..

Thx !

**Comment by DracoMilesX** [comment]
Check out JoshGlue branch of POGOserver and follow the setup steps. 

Currently the official branch doesn't support it.

**Comment by LUCKASSs** [comment]
Ok thx I will try

**Comment by LUCKASSs** [comment]
I can't find this branch.. @DracoMilesX

**Comment by DracoMilesX** [comment]
https://github.com/JoshGlue/POGOserver.git

**Comment by LUCKASSs** [comment]
@DracoMilesX This branch is outated.. Any tutorial on how to install Pogo-populate on 0.6.1 ?
@JoshGlue

**Comment by DracoMilesX** [comment]
Only way to do on 0.6.1 is to replace the files from this repo with the modified files from JoshGlue repo.

Otherwise you either need to check out the outdated repo or wait untill pull request is approved or another way is added.

**Comment by LUCKASSs** [comment]
That worked.
But I have an error when I log in game, so 0 Pokemons Spawns
`0 spawns created, maximum already reached`

@DracoMilesX

**Comment by DracoMilesX** [comment]
Sorry cannot help you with that one haven't tried pogo-populate myself yet.

**Comment by LUCKASSs** [comment]
@maierfelix @Cronick @JoshGlue

**Comment by Badley187** [comment]
don´t use pogo-populate... i tried it and its write every spawn to your mysql...  my db was full after after use this...

**Comment by LUCKASSs** [comment]
No problem for that, I can delete them
How do you install pogo populate on your server ? 
It give me this error :
`0 spawns created, maximum already reached`

**Comment by Badley187** [comment]
u need to use this https://github.com/JoshGlue/POGOserver/tree/c7f65a03d347c26e6f41ec477c7a9ef343217f05

clone it and install it, then test pogo-populate. but i say you delete it later

**Comment by LUCKASSs** [comment]
I'ts outdated

**Comment by Badley187** [comment]
yes, but the only version where pogo-populate worked.... to make pogo-popolutae work on 0.6.1 then you need to update pogo-populate ;)

**Comment by Badley187** [comment]
https://github.com/Badley187/POGOserver/tree/pogo-populate

try this...

**Comment by LUCKASSs** [comment]
EDIT : That worked with 
`sudo git clone --branch pogo-populate https://github.com/Badley187/POGOserver.git`

I can't
`sudo git clone https://github.com/Badley187/POGOserver/tree/pogo-populate` 
he said that he can't found this repo

Or `sudo git clone --branch pogo-populate git@github.com:Badley187/POGOserver.git`

It said 

```
Cloning into 'POGOserver'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

And how I can update the server by this branch ?

**Comment by LUCKASSs** [comment]
@Badley187 
Ok so I installed the server with your repo and your tree pogo populate but it gave me the same error ! 

`0 spawns created, maximum already reached`

**Comment by Badley187** [comment]
check your mysql spawn point table

i have tested it and its worked for me...

**Comment by LUCKASSs** [comment]
Where I can check that ?

**Comment by Badley187** [comment]
Phpmyadmin or Mysql Workbanch

**Comment by LUCKASSs** [comment]
Oh.. Right.. 600 page of spawns...... But no pokemon are spawning..

**Comment by Badley187** [comment]
//Populate map with Pokemon
  RANDOM_SPAWNS_AT_PLAYERS: true,
  RADIUS: 2, //In kilometers
  RARITY_RATE: 1.5, //The higher, the less likely rare Pokemon will appear
  NUMBER_SPAWNs: 500,
POKEMON_PER_SPAWN: 1,

change  RANDOM_SPAWNS_AT_PLAYERS to false
with true it create spawn point every time you login to your server.
i think you have to much spawn points...

**Comment by LUCKASSs** [comment]
Ok thx you so much 
I will try this

**Comment by LUCKASSs** [comment]
@Badley187 
Ok so, I have a clean database, a clean server with your pogopopulate tree, the server works fine without errors, but 0 pokemon spawns
And no message from PogoPopulate and nothing in the spawn_points

**Comment by Badley187** [comment]
Change RANDOM_SPAWNS_AT_PLAYERS to true, login with your player, wait some min. And check your db.then change RANDOM_SPAWNS_AT_PLAYERS to false,  log out and login again. Now the pokemons spawn.

**Comment by LUCKASSs** [comment]
@Badley187 
OK ! SO ! That works fine now !
BUT ... omg I can't support anymore .. Spawns are fine, but pokemon does not appear ! 
I just have this : http://image.noelshack.com/fichiers/2016/30/1469472074-screenshot-2016-07-23-16-18-45.png

**Comment by Badley187** [comment]
use your own api key, try to relog

**Comment by LUCKASSs** [comment]
I set my own Google map Api key but doesn't work..
@Badley187

**Comment by Badley187** [comment]
I dont know... i solved it with simple log out and log in.
